motionplan/tpspace/ptgGridSim.go

```diff
@@ -0,0 +1,234 @@
+package tpspace
+
+import (
+	"math"
+
+	"go.viam.com/rdk/spatialmath"
+)
+
+var (
+	defaultMaxTime       = 15.
+	defaultDiffT         = 0.005
+	defaultMinDist       = 3.
+	defaultAlphaCnt uint = 121
+	defaultTurnRad       = 100. // in mm, an approximate constant for estimating arc distances?
```

Comment:
This will be overwritten by whatever the frame turning radius is correct?

Reply:
Correct.

I'll add a TODO to do this as part of the following PR to finish connecting this to bases.